src/routes/api/request.routes.js

```diff
@@ -12,5 +12,7 @@ const {
 const router = express.Router();
 
 router.post('/', [isCurrentUserLoggedIn, isCurrentUserVerified, validateTripRequest], createTripRequest);
+router.get('/most-traveled-destinations', isCurrentUserLoggedIn, isCurrentUserVerified, placesAndVisitTimes);
+router.get('/accept-request/:id', acceptRequest);
```

Comment:
@descholar-ceo since you're updating the database, could you use 'PATCH' instead of 'GET' and use request body to hold the `id` field

Reply:
@karamuka I implemented this feedback